feat(integrations): Add troubleshooting docs for Vercel, JIRA and Azure DevOps (#3236)
getsentry/sentry-docs: src/docs/product/integrations/jira/index.mdx

## Troubleshooting

> ### Trouble with on premise installation

Reviewer: ```suggestion
### Trouble with Self-Hosted Sentry Installation
```

> If you're having trouble setting up Sentry with your on-premises integration, verify the following:

Reviewer: ```suggestion
If you're having trouble setting up Self-Hosted Sentry, verify the following:
```

- The provided installation URL is a fully qualified domain name (FQDN), which is resolvable on the internet.
- The IP addresses Sentry uses to make outbound requests are [allowed](/product/security/ip-ranges/).
- Sentry's access to your installation URL is not path restricted.

> ### "Something went wrong" message when installing JIRA on SaaS sentry.io

Reviewer: ```suggestion
### "Something went wrong" Message with JIRA on SaaS sentry.io Install
```

![](jira-something-went-wrong.png)

> This usually happens if you have an ad blocker that is blocking the conversation between JIRA and Sentry during setup. To remediate this, disable your ad blocker and go through the installation flow again.

Reviewer: ```suggestion
This issue typically occurs if you have an ad blocker blocking the conversation between JIRA and Sentry during setup. To remediate this, disable your ad blocker and go through the installation flow again.
```
